src/main/java/tds/testpackageconverter/converter/impl/TestPackageConverterServiceImpl.java

```diff
@@ -73,6 +74,14 @@ public boolean convertTestSpecifications(final String testPackageName, final Lis
         return convertedTestPackageFile.createNewFile();
     }
 
+    @Override
+    public boolean convertTestPackage(final TestPackage testPackage) {
+        List<Testspecification> administrationPackages = LegacyAdministrationTestPackageMapper.fromNew(testPackage);
+
```

Comment:
This is just a stub for now. This implementation method (and the interface documentation) will be added in as part of TDS-1614.